cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiImpl.java

```diff
@@ -285,6 +286,7 @@ public Future<Void> resume(String host, int port, String connectorName) {
     }
 
     private Future<Void> pauseResume(String host, int port, String path) {
+
```

Comment:
I don't think we need this empty line here. Should we add the log message instead here?

Reply:
Agree

Reply:
Hi, will add a logger there also ( in `pause` and `resume` ). But Logger requires `reconciliation` as a parameter so need to change the signature of pause and resume as instructed by @kyguy. If you have anything to suggest please let me know.
Will push the code soon.

Reply:
@omjikush09 Looking solid, great work! Just address the checkstyle errors and make sure the unit tests are passing